src/main/java/de/sakpaas/backend/service/LocationDetailsRepository.java

```diff
@@ -2,10 +2,15 @@
 
 import de.sakpaas.backend.model.LocationDetails;
 import java.util.Optional;
+import java.util.Set;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
 
 public interface LocationDetailsRepository extends JpaRepository<LocationDetails, Long> {
 
   Optional<LocationDetails> findById(Long id);
 
+  @Query(value = "select lower(brand) from location_details where brand is not null group by brand",
```

Comment:
Ich finde es schöner, wenn die SQL commands (`SELECT`) in caps geschrieben sind, damit sie leichter von Variablen/Feldern unterscheiden kann. Macht das meiner Meinung nach insgesamt lesbarer.

Reply:
Außerdem gibt es einen Fehler in der Abfrage, die Elemente müssten nach `lower(brand)` gegroupt sein. (Ohne Fix sind es bei mir 279 Einträge und mit Fix sind es 234).
`SELECT lower(brand) FROM location_details WHERE brand IS NOT null GROUP BY lower(brand);`

Reply:
Geändert